src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -293,15 +294,17 @@ def tree_values_preprocessing(
 # pylint: disable=too-many-locals
 def tree_to_numpy(
     model: Callable,
-    x: numpy.ndarray,
+    q_x: numpy.ndarray,
```

Comment:
actually, this is an API breaking change no ? we should probably avoid this, but create an issue to note this change and add it to https://github.com/zama-ai/concrete-ml-internal/issues/3995